### ISSUE TITLE
i cann't set margin

### ISSUE THREAD

**Issue by aduonggm** [kozakdenys/qr-code-styling#45]

**Comment by awran5** [comment]
I'm facing the same issue. It seems that the `margin` property is just missing from `Options` declaration at 
```qr-code-styling/lib/core/QROptions``` file.

As a workaround I did:

```ts
import { Options } from 'qr-code-styling/lib/core/QROptions'

interface MissingOption extends Options {
  margin?: number
}
...

const qrCode = new QRCodeStyling({
  width: ...,
  height: ...,
  margin: 10, // Add missing margin
  image: ...,
  dotsOptions: {},
  imageOptions: {},
} as MissingOption) // add extended interface

```

**Comment by kozakdenys** [comment]
@aduonggm @davemecha @awran5 thanks for your help
the bug was fixed in [1.4.2](https://github.com/kozakdenys/qr-code-styling/releases/tag/v1.4.2) varsion
